content/showcase/_index.html

```diff
@@ -12,10 +12,18 @@ <h1 class="title is-2">UQCS Showcase</h1>
         <div class="columns is-centered">
             <div class="column is-8">
                 <div class="showcase-gallery columns is-multiline">
+                    {{< showcase-item
+                        src="/img/media/showcase/codeduel.png"
+                        title="CodeDuel"
+                        caption="<p> Bored of Leetcoding by yourself? Well, challenge a friend to see who's better! <br> Oh, did we mention you can <b> sabotage </b> them and watch them lose their minds? 🤯 </p>"
+                        image-class="showcase-image"
+                        link="https://github.com/Antimatter543/CodeClash"
+                        class="showcase-element has-text-centered"
+                    >}}
                     {{< showcase-item
                         src="/img/media/showcase/imvim.jpg"
                         title="imvim"
-                        caption="<p>\"Improved\" Vi Improved: the <s>cursed</s> (efficient and intuitive!!) text-editor. Created using Tkinter.</p><p><b>2024 Hackathon Most Masochistic</b></p>"
+                        caption="<p>\'Improved\' Vi Improved: the <s>cursed</s> (efficient and intuitive!!) text-editor. Created using Tkinter.</p><p><b>2024 Hackathon Most Masochistic</b></p>"
```

Comment:
This seems to break the quotation marks for VI Improved.

Reply:
I forgot to change that back - hugo was erroring on me locally since it couldn't interpret the \ as an escape for some reason, so I temp changed it. Will fix now!